documentatie/use-cases/gsb-invoer-eerste-zitting.md

```diff
@@ -218,25 +218,23 @@ Voor elke foutmelding:
 
 __Uitbreidingen:__  
 2a. De invoerder stelt een fout op het PV vast en kan de foutmelding niet oplossen:  
-&emsp; 2a1. De invoerder meldt de fout op het PV bij de coördinator GSB.  
-&emsp; 2a2. De coördinator GSB stuurt het PV terug in het proces.  
-&emsp; 2a3. De invoerder breekt de invoer af.  
-&emsp; 2a4. De applicatie verwijdert de ingevoerde data.  
+&emsp; 2a1. De invoerder meldt de fout op het PV bij de coördinator GSB.
+&emsp; 2a2. De invoerder kan in overleg het volledige PV invoeren.
+&emsp; 2a3. De coördinator GSB beoordeelt het PV en het overzicht van fouten. 
+&emsp; 2a4. De coördinator GSB stuurt het PV terug in het proces.
+&emsp; 2a5. De applicatie verwijdert de ingevoerde data.  
```

Comment:
De formatting gaat hier niet goed (mist twee spaties aan het einde van iedere regel voor line breaks in Markdown)